src/components/InputField.js

```diff
@@ -9,6 +9,7 @@ import InputTypeLocation from './InputTypeLocation';
 import InputTypeCreditCard from './InputTypeCreditCard';
 import InputSwitch from './InputSwitch';
 import InputTypeCountry from './InputTypeCountry';
+import ArchiveCollective from './ArchiveCollective';
```

Comment:
Can we avoid having `ArchiveCollective` in `InputField`, what does that bring us?